frontend/packages/data-portal/app/utils/url.test.ts

```diff
@@ -8,8 +8,8 @@ describe('utils/url', () => {
       expect(isExternalUrl('https://www.google.com')).toBe(true)
     })
 
-    it('should return true for non-external URLs', () => {
-      expect(isExternalUrl('/faq')).toBe(false)
+    it('should return true for external URLs', () => {
+      expect(isExternalUrl('https://chanzuckerberg.github.io/cryoet-data-portal/cryoet_data_portal_docsite_faq.html')).toBe(true)
```

Comment:
You were right that the linter did not like this long line! 

However I think this test is meant to check that the isExternalUrl function is working so let's change it to this:

```suggestion
   it('should return true for non-external URLs', () => {
      expect(isExternalUrl('/competition')).toBe(false)
    })
```